docarray/array/mixins/io/pushpull.py

```diff
@@ -11,44 +11,26 @@
 if TYPE_CHECKING:
     from docarray.typing import T
 
-JINA_CLOUD_CONFIG = 'config.json'
-
 
 @lru_cache()
```

Comment:
It is dangerous to use @lru cache on the token since the token can change. We revert it recently see this pr for more information https://github.com/jina-ai/docarray/pull/469